docs-v2/reference/scripts.mdx

```diff
@@ -56,37 +56,343 @@ Read more about [integration scripts](/understand/concepts/scripts) to understan
 
 Integration scripts expose a helper object (`NangoSync` for sync scripts, `NangoAction` for action scripts), which allows to interact with external APIs & Nango more easily. 
 
+# HTTP requests
+
 <Tip>
-The script helper object shares some methods, but not all, with the [Node backend SDK](/reference/sdks/node), which is why this reference will link to the Node SDK reference a lot.
+Note that all HTTP requests benefit from automatic credential injection. Because scripts are executed in the context of a specific integration & connection, Nango can automatically retrieve  & refresh the relevant API credentials.
 </Tip>
 
-# Common helper methods
+### GET requests
 
-### HTTP requests
+Makes an HTTP GET request inside an integration script.
 
-HTTP requests in integration scripts are made with the proxy ([concept]((/understand/concepts/proxy)) / [step-by-step guide](/integrate/guides/proxy-requests-to-an-api) / [reference](/reference/sdks/node#proxy)).
+```js
+const res = await nango.get({
+    endpoint: '/endpoint',
+    baseUrlOverride: 'https://base-url.com'
```

Comment:
why do we mention baseUrlOverride (here and in the example below)? It is a optional param that is defined below. I would keep the example as simple as possible imho

Reply:
Done!